pkgs/games/blackvoxel/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, glew110, SDL
+}:
+
+stdenv.mkDerivation {
+  pname = "blackvoxel";
+  version = "2.50";
+
+  buildInputs = [ glew110 SDL ];
+
+  src = fetchFromGitHub {
+    owner = "Blackvoxel";
+    repo = "Blackvoxel";
+    rev = "2.5";
+    hash = "sha256-Uj3TfxAsLddsPiWDcLKjpduqvgVjnESZM4YPHT90YYY=";
+  };
+
+  buildFlags = [ "BV_DATA_LOCATION_DIR=$(out)/data" ];
+
+  installFlags = [ "doinstall=true BV_DATA_INSTALL_DIR=$(out)/data BV_BINARY_INSTALL_DIR=$(out)/bin" ];
```

Comment:
```suggestion

  src = fetchFromGitHub {
    owner = "Blackvoxel";
    repo = "Blackvoxel";
    rev = "2.5";
    hash = "sha256-Uj3TfxAsLddsPiWDcLKjpduqvgVjnESZM4YPHT90YYY=";
  };

  buildFlags = [ "BV_DATA_LOCATION_DIR=$(out)/data" ];

  buildInputs = [ glew110 SDL ];

  installFlags = [
    "doinstall=true BV_DATA_INSTALL_DIR=$(out)/data BV_BINARY_INSTALL_DIR=$(out)/bin"
  ];
```

Reply:
Is this convention written anywhere ? I've read some parts of the nixpkgs manual and the CONTRIBUTING file.